metricbeat/docs/fields.asciidoc

```diff
@@ -19633,37 +19633,37 @@ Total number of rows returned by query.
 
 --
 
-*`postgresql.statement.time.total`*::
+*`postgresql.statement.query.time.total.ms`*::
```

Comment:
Should this be `postgresql.statement.query.time.total` instead of `total.ms`? I see in https://github.com/elastic/beats/blob/master/metricbeat/module/postgresql/statement/_meta/data.json#L27, it doesn't have the `.ms` part.